docs/sql-data-sources-parquet.md

```diff
@@ -286,6 +286,24 @@ Data source options of Parquet can be set via:
     </td>
     <td>read</td>
   </tr>
+  <tr>
+    <td><code>mergeSchema</code></td>
+    <td>The SQL config <code>spark.sql.parquet.mergeSchema</code> which is <code>false</code> by default.</td>
+    <td>Sets whether we should merge schemas collected from all Parquet part-files. This will override <code>spark.sql.parquet.mergeSchema</code>.</td>
+    <td>read</td>
+  </tr>
+  <tr>
+    <td><code>compression</code></td>
+    <td>None</td>
+    <td>Compression codec to use when saving to file. This can be one of the known case-insensitive shorten names (none, uncompressed, snappy, gzip, lzo, brotli, lz4, and zstd). This will override <code>spark.sql.parquet.compression.codec</code>. If None is set, it uses the value specified in <code>spark.sql.parquet.compression.codec</code>.</td>
+    <td>write</td>
+  </tr>
+  <tr>
+    <td><code>partitionBy</code></td>
+    <td>None</td>
+    <td>Names of partitioning columns</td>
+    <td>write</td>
+  </tr>
```

Comment:
Can we point out an URL link for the general options like `pathGlobFilter` and/or `modifiedBefore`?